lake/ztests/seek-index.yaml

```diff
@@ -1,14 +1,22 @@
 script: |
-  zed lake import -streammax=100 -asc -R asc babble.zson
-  zed lake query -z -start 2020-04-21T23:59:26.063Z -end 2020-04-21T23:59:38.069Z -s -R asc "*"
+  export ZED_LAKE_ROOT=test
+  zed lake init -q
+  zed lake create -p asc -order asc -q
+  zed lake load -q -streammax=100 -p asc babble.zson
+  zed lake query -z -start 2020-04-21T23:59:26.063Z -end 2020-04-21T23:59:38.069Z -s -p asc "*"
   echo === | tee /dev/stderr
-  zed lake import -streammax=100 -R desc babble.zson
-  zed lake query -z -start 2020-04-21T23:59:26.063Z -end 2020-04-21T23:59:38.069Z -s -R desc "*"
+  zed lake create -p desc -order desc -q
+  zed lake load -q -streammax=100 -p desc babble.zson
+  zed lake query -z -start 2020-04-21T23:59:26.063Z -end 2020-04-21T23:59:38.069Z -s -p desc "*"
 
 inputs:
   - name: babble.zson
     source: ../../testdata/babble.zson
 
+# Note: compared to the previous zar model, we do not have aritificial
+# partitions into days so in this sample data, we read a whole seek index
+# which is larger compared to the day.
+
```

Comment:
While this is useful context for reviewers, I don't think it'll be interesting to future test readers, so I wouldn't commit it.

Reply:
I will take out the reference to the previous approach but the comment makes a non-obvious point that I spent time rediscovering because I introduced a bug that broke the old test.